helpers/middleman/landing-pages.rb

```diff
@@ -22,9 +22,11 @@ def landing_pages_collection(locale_obj: default_locale_obj, app_data: data)
 
     @landing_pages_collection ||= {}
     @landing_pages_collection[lang] ||= app_data['landing-pages'].pages.map do |tuple|
-      model = tuple[1]
+      model = tuple[1] # contentful passes ["id", { ... }]
       localized_model = localize_entry(model, lang, default_locale_obj[:lang])
       localized_model[:title] = landing_page_title(localized_model, locale_obj: locale_obj)
+      localized_model[:description] = localized_model[:page_description]
```

Comment:
Oh, I see.  We are looking for `locals[:description]`, not `locals[:page_description]`.

👍